frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/review-tab/general-tab.tsx

```diff
@@ -46,9 +47,11 @@ const GeneralReviewConnected: React.FC<GeneralReviewConnectedProps> = (props) =>
 
       <FormFieldReviewMemoRow field={getField(VMSettingsField.DESCRIPTION, iVMSettings)} />
 
-      <FormFieldReviewMemoRow
-        field={getField(VMSettingsField.PROVISION_SOURCE_TYPE, iVMSettings)}
-      />
+      {!isImport && (
+        <FormFieldReviewMemoRow
```

Comment:
now we need the keys :) 

practically no - because this value will not change during the wizard instance, but it is a good practice to specify it in places like this.

Reply:
Done!